### PR TITLE
[Android] display bookmark export file path

### PR DESCRIPTION
**Pull request by tapanmodh**
Resolves https://github.com/brave/brave-browser/issues/33171

## Submitter Checklist:

- [x] I confirm that no [security/privacy review is needed](https://github.com/brave/brave-browser/wiki/Security-reviews) and no other type of reviews are needed, or that I have [requested](https://github.com/brave/reviews/issues/new/choose) them
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [x] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

## Test Plan:
1. Go to bookmarks
2. Tap on three dot menu
3. Select Export bookmarks
4. You can see file path as well as `Open folder` button to open the folder containing exported bookmarks

### CODE REVIEW

**Review by deeppandya — APPROVED** [review]
LGTM

File: browser/ui/android/strings/android_brave_strings.grd [hunk 1]
```diff
@@ -1278,17 +1278,32 @@ This file contains all "about" strings.  It is set to NOT be translated, in tran
       <message name="IDS_IMPORT_BOOKMARKS_SELECT_FILE" desc="Import Bookmarks file chooser text">
         Select a File to Import bookmarks
       </message>
-      <message name="IDS_IMPORT_BOOKMARKS_SUCCESS" desc="Import Bookmarks success text message.">
+      <message name="IDS_EXPORT_BOOKMARKS_SUCCESS_TITLE" desc="Export Bookmarks success title text.">
+        Export successful
```
Comment (mkarolin) [inline browser/ui/android/strings/android_brave_strings.grd:1282]: Suggestion: `Export succeeded` (to match the opposite `Export failed`).

File: browser/ui/android/strings/android_brave_strings.grd [hunk 2]
```diff
@@ -1278,17 +1278,32 @@ This file contains all "about" strings.  It is set to NOT be translated, in tran
...
+      </message>
+      <message name="IDS_EXPORT_BOOKMARKS_SUCCESS_BODY" desc="Export Bookmarks success text message.">
+        Bookmarks exported to <ph name="EXPORT_FILE_PATH">%1$s<ex>/Download/bookmarks.html</ex></ph>
```
Comment (mkarolin) [inline browser/ui/android/strings/android_brave_strings.grd:1285]: Suggestion: `Exported bookmarks to <ph...`

File: browser/ui/android/strings/android_brave_strings.grd [hunk 3]
```diff
@@ -1278,17 +1278,32 @@ This file contains all "about" strings.  It is set to NOT be translated, in tran
...
+      </message>
+      <message name="IDS_EXPORT_BOOKMARKS_FAIL_TITLE" desc="Export Bookmarks fail title text.">
+        Export failed
+      </message>
+      <message name="IDS_EXPORT_BOOKMARKS_FAIL_BODY" desc="Export Bookmarks fail text message.">
+        Bookmarks export failed. Please try again.
```
Comment (mkarolin) [inline browser/ui/android/strings/android_brave_strings.grd:1291]: Suggestion: `Failed to export bookmarks...`

File: browser/ui/android/strings/android_brave_strings.grd [hunk 4]
```diff
@@ -1278,17 +1278,32 @@ This file contains all "about" strings.  It is set to NOT be translated, in tran
...
+      </message>
+      <message name="IDS_IMPORT_BOOKMARKS_SUCCESS_TITLE" desc="Import Bookmarks success title text.">
+        Import successful
```
Comment (mkarolin) [inline browser/ui/android/strings/android_brave_strings.grd:1294]: Suggestion: `Export succeeded`

File: browser/ui/android/strings/android_brave_strings.grd [hunk 5]
```diff
@@ -1278,17 +1278,32 @@ This file contains all "about" strings.  It is set to NOT be translated, in tran
...
+      </message>
+      <message name="IDS_IMPORT_BOOKMARKS_SUCCESS_BODY" desc="Import Bookmarks success text message.">
         Bookmarks imported successfully.
```
Comment (mkarolin) [inline browser/ui/android/strings/android_brave_strings.grd:1297]: Suggestion: Imported bookmarks successfully.

File: browser/ui/android/strings/android_brave_strings.grd [hunk 6]
```diff
@@ -1278,17 +1278,32 @@ This file contains all "about" strings.  It is set to NOT be translated, in tran
...
       </message>
-      <message name="IDS_IMPORT_BOOKMARKS_FAILED" desc="Import Bookmarks failed text message.">
-        Bookmark import failed.
+      <message name="IDS_IMPORT_BOOKMARKS_FAIL_TITLE" desc="Import Bookmarks fail title text.">
+        Import failed
       </message>
-      <message name="IDS_EXPORT_BOOKMARKS_FAILED" desc="Export Bookmarks failed text message.">
-        Bookmark export failed.
+      <message name="IDS_IMPORT_BOOKMARKS_FAIL_BODY" desc="Import Bookmarks fail text message.">
+        Bookmark import failed. Please try again.
```
Comment (mkarolin) [inline browser/ui/android/strings/android_brave_strings.grd:1303]: Suggestion:  Failed to import bookmark(s?)...

**Review by mkarolin — APPROVED** [review]
`strings`++ (with suggestions).

File: browser/ui/android/strings/android_brave_strings.grd (hunk 1 above)
Comment (tapanmodh) [reply browser/ui/android/strings/android_brave_strings.grd:1282]: @anthonypkeane wdyt?

**Pull request merged**